### PR TITLE
docs(configuration options): style fixes for `prPriority`

### PR DESCRIPTION
**Pull request by HonkingGoose**
## Changes

- Monospace the update types, and fix a typo (`patches` should be `patch` update type)
- Monospace the `devDependencies` term

## Context

Drive-by PR, not closing any issue with this.

## Documentation (please check one with an [x])

- [x] I have updated the documentation, or
- [ ] No documentation update is required

## How I've tested my work (please select one)

I have verified these changes via:

- [x] Code inspection only, or
- [ ] Newly added/modified unit tests, or
- [ ] No unit tests but ran on a real repository, or
- [ ] Both unit tests + ran on a real repository

### CODE REVIEW

File: docs/usage/configuration-options.md [hunk 1]
```diff
@@ -2765,11 +2765,11 @@ This is why we configured an upper limit for how long we wait until creating a P
 ## prPriority
 
 Sometimes Renovate needs to rate limit its creation of PRs, e.g. hourly or concurrent PR limits.
-In such cases it sorts/prioritizes by default based on the update type (e.g. patches raised before minor, minor before major).
+In such cases it sorts/prioritizes by default based on the update type (e.g. `patch` raised before `minor`, `minor` before `major`).
```
Comment (HonkingGoose) [inline docs/usage/configuration-options.md:2768]: This line feels clunky.

If I'm reading this right, if Renovate is rate-limited it prioritizes updates in this order: 

1. `patch`
2. `minor`
3. `major`

I expected it to be like this, before I read the docs:

1. `major`
2. `minor`
3. `patch`

I guess the philosophy here is: "Give them the easy updates first and worry about the big stuff later, when I'm no longer rate-limted."?

Comment (rarkins) [reply docs/usage/configuration-options.md:2768]: Yes, easiest stuff first. Also it prioritizes regardless of whether it's rate limited or not

Comment (HonkingGoose) [reply docs/usage/configuration-options.md:2768]:
```suggestion
By default, Renovate sorts/prioritizes based on the update type, going from smallest update to biggest update in order.
So Renovate will create `patch` updates first, then `minor` updates, and finally `major `updates.
```

How about this then?

Comment (rarkins) [reply docs/usage/configuration-options.md:2768]: Yes, but actually it goes pinDigest, pin, digest, patch, minor, major

Comment (HonkingGoose) [reply docs/usage/configuration-options.md:2768]:
```suggestion
By default, Renovate sorts/prioritizes based on the update type, going from smallest update to biggest update.
Renovate creates update PRs in this order:

1. `pinDigest`
1. `pin`
1. `digest`
1. `patch`
1. `minor`
1. `major`

```

**Review by rarkins — APPROVED** [review]

**Pull request merged**